test.sh

```diff
@@ -36,6 +36,9 @@ k_wait_all_running() { bash ci/wait-pods-ok; }
     sleep 20
     kubectl describe VirtualMachineInstanceMigration testvm-migration
   fi
+
+  kubectl delete -n kubevirt kubevirt kubevirt
```

Comment:
can you change this line to

```
kubectl delete -n kubevirt kubevirt kubevirt --wait=true
```

Reply:
Will do, I doubt it helps.

Reply:
I suppose we need to add a line which explicitly deletes the webhook registraion before deleting the namespace.

Reply:
While it is not a 100% clean uninstall, the apiserver registration should just be deleted when the namespace gets deleted :/ We don't have a finalizer on it.

Reply:
Ah wait, they are not even namespaced. They are left behind but should not block namespace deletions ...